config/flags_payments.go

```diff
@@ -65,16 +65,28 @@ var (
 		Name:  "payments.disable",
 		Usage: "Disables payments and moves to a backwards compatible legacy mode",
 	}
-	// FlagPaymentsConsumerUpperPriceBound sets the upper price bound to a set value.
-	FlagPaymentsConsumerUpperPriceBound = cli.Uint64Flag{
-		Name:  "payments.consumer.price.upper.bound",
-		Usage: "Sets the maximum price of the service. All proposals with a price above this bound will be filtered out and not visible.",
-		Value: 1000000,
+	// FlagPaymentsConsumerPricePerMinuteUpperBound sets the upper price bound per minute to a set value.
+	FlagPaymentsConsumerPricePerMinuteUpperBound = cli.Uint64Flag{
+		Name:  "payments.consumer.price.perminute.upper.bound",
+		Usage: "Sets the maximum price of the service per minute. All proposals with a price above this bound will be filtered out and not visible.",
+		Value: 50000,
 	}
-	// FlagPaymentsConsumerLowerPriceBound sets the lower price bound to a set value.
-	FlagPaymentsConsumerLowerPriceBound = cli.Uint64Flag{
-		Name:  "payments.consumer.price.lower.bound",
-		Usage: "Sets the minimum price of the service. All proposals with a below above this bound will be filtered out and not visible.",
+	// FlagPaymentsConsumerPricePerMinuteLowerBound sets the lower price bound per minute to a set value.
+	FlagPaymentsConsumerPricePerMinuteLowerBound = cli.Uint64Flag{
+		Name:  "payments.consumer.price.perminute.lower.bound",
```

Comment:
Similarly, 
```suggestion
		Name:  "payments.consumer.price-perminute-min",
```
And so on.